spec/factories/courses.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :course do
-    association :teacher, factory: [:user, :teacher]
+    association :teacher, factory: [:user, :teacher], email: "somerandomemail@r.com", google_id: "sdkfjdsl2342342"
```

Comment:
Just a personal preference, but might be good to make the string itself indicate that it's a google_id: "google_id123" for instance.